src/main/java/seedu/letsgethired/ui/UtilityButtonBar.java

```diff
@@ -0,0 +1,41 @@
+package seedu.letsgethired.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.layout.Region;
+import seedu.letsgethired.logic.commands.exceptions.CommandException;
+import seedu.letsgethired.logic.parser.exceptions.ParseException;
+
+/**
+ * A ui for utility buttons below the select view
+ */
+public class UtilityButtonBar extends UiPart<Region> {
```

Comment:
Perhaps you could consider renaming this class to show more clearly that it's coupled to an `InternApplicationListPanel`? 
Something like `InternApplicationUtilityButton`?

Just a suggestion!